src/accounts/decoders.rs

```diff
@@ -71,4 +82,19 @@ mod tests {
             assert!(false, "error decoding position: {err}");
         }
     }
+
+    #[test]
+    fn decode_family_codes() {
+        let mut message = super::ResponseMessage::from("0DU1236109\0F445566");
```

Comment:
How did you get the sample response message? I usually start by creating an example for the API call. (e.g. https://github.com/wboayue/rust-ibapi/blob/main/examples/contract_details.rs). When we request is encoded correctly I can look at the logs to get a sample of the response message.

Reply:
I am stuck at this test.  How do I get the logs from the sample of the response message?
I've created a family_codes example: 
fn main() -> anyhow::Result<()> {
    let client = Client::connect("127.0.0.1:4002", 100).expect("connection failed");
 
    let family_codes = client.family_codes().expect("request failed");
    for family_code in family_codes {
        println!("{:4} {:4}", family_code[0].account_id, family_code[0].family_code)
    }

    Ok(())

}

Reply:
I am stuck at this problem.  Can you show me how you got the logs from 'positions' response message?

Reply:
@JoeTonDev for this part you need an interactive broker paper trading account. With the IB desktop or gateway client running, I would create and example that uses your new code to fetch the family codes.

You can use [this example](https://github.com/wboayue/rust-ibapi/blob/main/examples/head_timestamp.rs) as a guide.

Notice the call to `env_logger::init`. This allows you to turn logging on/off using an environment variable. So you can go this to see the messages being sent and received from the IB server.

```bash
RUST_LOG=debug cargo run --example family_codes
```